### DISCUSSION TITLE
怎么设置，不需要登录，游客就可以访问主页

### DISCUSSION THREAD

**Discussion by pmi235** [AlistGo/alist#6319] [Q&A]
怎么设置，不需要登录，游客就可以访问主页

**Comment by xavierskip** [comment]
新版默认关闭了游客访问，需要自行手动开启

[https://alist.nn.ci/zh/guide/advanced/user.html#tips](https://alist.nn.ci/zh/guide/advanced/user.html#tips)

Guest user is disabled, login please：为了保护您的Alist安全，游客访问权限关闭了，若需要游客访问自行打开

后台 --> 用户 --> guest --> 停用取消勾选
